security/hydra/Portfile

```diff
@@ -19,82 +20,36 @@ long_description \
     support.
 
 homepage                https://github.com/vanhauser-thc/thc-hydra
```

Comment:
The `homepage` will already be set to this by the `github.setup` command. So this line should be removed.